dev/ci/ci-user-overlay.sh

```diff
@@ -29,3 +30,8 @@ if [ $TRAVIS_PULL_REQUEST == "461" ] || [ $TRAVIS_BRANCH == "stm+remove_compat_p
     mathcomp_CI_BRANCH=no_camlp4_compat
     mathcomp_CI_GITURL=https://github.com/ejgallego/math-comp.git
 fi
+
+if [ $TRAVIS_PULL_REQUEST == "565" ] || [ $TRAVIS_BRANCH == "remove-vernacerror" ]; then
+    mathcomp_CI_BRANCH=remove-vernacerror
+    mathcomp_CI_GITURL=https://github.com/SkySkimmer/math-comp.git
```

Comment:
Right but notice that the code for the `git_checkout` function in `dev/ci/ci-common.sh` already does this in two steps:

```
  # Allow an optional 4th argument for the commit
  local _COMMIT=${4:-FETCH_HEAD}
  local _DEPTH=$(if [ -z "${4}" ]; then echo "--depth 1"; fi)

  mkdir -p ${_DEST}
  ( cd ${_DEST}                                                && \
    if [ ! -d .git ] ; then git clone ${_DEPTH} ${_URL} . ; fi && \
    echo "Checking out ${_DEST}"                               && \
    git fetch ${_URL} ${_BRANCH}                               && \
    git checkout ${_COMMIT}                                    && \
    echo "${_DEST}: `git log -1 --format='%s | %H | %cd | %aN'`" )
```

and for almost all developments `${_COMMIT}` will be `FETCH_HEAD`.

I just tested this on my own machine:

    mkdir foo
    git init
    git fetch https://github.com/math-comp/math-comp refs/pull/120/head
    git checkout FETCH_HEAD

and it seemed to work fine.